docs/apache-airflow-providers-ftp/index.rst

```diff
@@ -86,4 +85,19 @@ You can install this package on top of an existing Airflow 2 installation (see `
 for the minimum Airflow version supported) via
 ``pip install apache-airflow-providers-ftp``
 
+Requirements
```

Comment:
cc: @eladkal - found a small bug here - the whole "Requirements" section was excluded completely when the package had no `pip` requirements at all (this happened to sftp provider). It was fine before, because technically it had no requirements (it was a preinstalled provide), but now at the very least there is airflow 2.4.0 as requirements, so the whole chapter is needed always.